src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -77,6 +77,18 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
       this.stopPolling();
 
+      // Wait 1 min before polling again & do not show error message when encounter rate limit error
+      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
+        error.responseJSON?.errorCode === 'E0000047') {
+        setTimeout(() => {
+          model.trigger('clearFormError');
+        }, 1);
```

Comment:
Any reason why you wanted `1` here instead of `0` ? `0` basically implies as soon as possible.

Reply:
Sure let me update 0. I thought I need a little delay to make sure that the error is processed first. Tested with 0 and it work.